### PR TITLE
Flag models we cannot calculate cost

### PR DESCRIPTION
**Pull request by maamalama**
Flag models we cannot calculate cost

### CODE REVIEW

File: web/components/templates/requestsV2/costPill.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  const handleOpenModal = (
+    event: React.MouseEvent<HTMLSpanElement, MouseEvent>
+  ) => {
+    event.stopPropagation();
+    setIsOpen(true);
+  };
+
+  return (
+    <>
+      <span
```
Comment (ScottMktn) [inline web/components/templates/requestsV2/costPill.tsx:19]: nit: span's should not have an onClick one them. Change this to a button and make sure the flex is inline.

File: web/components/templates/requestsV2/costPill.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        className={clsx(
+          "bg-gray-50 text-gray-700 ring-gray-200 dark:bg-gray-900 dark:text-gray-300 dark:ring-gray-800",
+          "w-max flex items-center rounded-lg px-2 py-1 -my-1 text-xs font-medium ring-1 ring-inset cursor-pointer"
+        )}
+      >
+        Cost Unsupported{" "}
+        <ArrowTopRightOnSquareIcon className="ml-1 h-4 w-4 text-orange-500 inline-block" />
+      </span>
+
+      {isOpen && (
```
Comment (ScottMktn) [inline web/components/templates/requestsV2/costPill.tsx:30]: no need to add the `isOpen` check here. the ThemedModal will handle this for you. You'll notice a smoother animation once you remove this.

File: web/components/templates/requestsV2/costPill.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        <ThemedModal open={isOpen} setOpen={() => setIsOpen(false)}>
+          <div
```
Comment (ScottMktn) [inline web/components/templates/requestsV2/costPill.tsx:32]: see above. This should be a `<button>`

File: web/components/templates/requestsV2/costPill.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    event.stopPropagation();
+    setIsOpen(true);
+  };
+
+  return (
+    <>
+      <span
+        onClick={handleOpenModal}
+        className={clsx(
+          "bg-gray-50 text-gray-700 ring-gray-200 dark:bg-gray-900 dark:text-gray-300 dark:ring-gray-800",
```
Comment (ScottMktn) [inline web/components/templates/requestsV2/costPill.tsx:22]: are we going with orange or gray? I'm okay with either - but what do you think?

File: web/components/templates/requestsV2/costPill.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+          >
+            <h1 className="col-span-4 font-semibold text-xl text-gray-900 dark:text-gray-100">
+              Cost Unsupported
+            </h1>
+
+            <p className="text-sm text-gray-900 dark:text-gray-100">
+              Currently, we do not support the cost of this model. Please create
+              an issue in Github{" "}
+              <Link
+                target="_blank"
```
Comment (ScottMktn) [inline web/components/templates/requestsV2/costPill.tsx:44]: add `rel=noopener noreferrer` to `<Link>` components whenever you are opening a new tab

File: web/components/templates/requestsV2/costPill.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+                href="https://discord.gg/zsSTcH2qhG"
+                className="underline text-blue-500"
+              >
+                https://discord.gg/zsSTcH2qhG
+              </Link>{" "}
+            </p>
+            <div className="col-span-4 flex justify-end gap-2 pt-4">
+              <button
+                type="submit"
```
Comment (ScottMktn) [inline web/components/templates/requestsV2/costPill.tsx:61]: make this a type `button` and add an explicit onClick here that closes the modal.

File: web/components/templates/requestsV2/costPill.tsx (hunk 4 above)
Comment (maamalama) [reply web/components/templates/requestsV2/costPill.tsx:22]: @ScottMktn orange looks good and noticeable, I like it

**Pull request merged**